lib/airtable/table.rb

```diff
@@ -27,6 +27,28 @@ def records(options={})
       RecordSet.new(results)
     end
 
+    # Query for records using a string formula
+    # Options: limit = 100, offset = "as345g", sort = ["Name", "asc"],
+    #          fields = [Name, Email], formula = "Count > 5", view = "Main View"
+    #
+    # select(limit: 10, sort: ["Name", "asc"], formula: "Order < 2")
+    def select(options={})
+      options['sortField'], options['sortDirection'] = options.delete(:sort) if options[:sort]
+      options['maxRecords'] = options.delete(:limit) if options[:limit]
+
+      if options[:formula]
+        raise_bad_formula_error unless options[:formula].is_a? String
+        options['filterByFormula'] = options.delete(:formula)
+      end
+
+      results = self.class.get(worksheet_url, query: options).parsed_response
+      RecordSet.new(results)
+    end
```

Comment:
The `all` method does support aggregating between all results (https://github.com/nesquena/airtable-ruby/blob/master/lib/airtable/table.rb#L7-L19). Unfortunately, I am not actively maintaining this but I've added you both as contributors to the repo and would be happy to add you the ability to cut gems as well. I'd love to find people willing to take this over, I'm happy to move this over to the airtable repo respository when/if it makes sense.